icc-x-api/storage/LocalStorageFacade.ts

```diff
@@ -0,0 +1,52 @@
+export interface LocalStorageFacade {
+
+  /**
+   * Returns the value of the provided key from the storage
+   * @param key Key of the value to retrieve
+   * @return The value associated to the provided key or null if not found.
+   */
+  getItem(key: string): string | null
```

Comment:
Return `string?` instead of `string | null`. In Typescript, prefer to return undefined values instead of null, they are better managed